app/_src/gateway/kong-enterprise/cp-outage-handling.md

```diff
@@ -0,0 +1,142 @@
+---
+title: How to Configure Data Plane Resilience
+badge: enterprise
+content_type: how-to
+---
+
+Starting in {{site.base_gateway}} version 3.2, {{site.base_gateway}} can be configured to support configuring new data planes in the event of a control plane outage. This feature works by designating a backup node, and allowing it read/write access to a data store. This backup node will automatically push valid {{site.base_gateway}} configurations to the data store. In the event of a control plane outage, if a new node is created, it will pull the latest {{site.base_gateway}} configuration from the data store, configure itself, and start proxying requests. 
+
+This option is only recommended for customers who are have to adhere to strict availability SLAs, because it requires a larger maintenance load. 
+
+{% navtabs %}
+{% navtab Amazon S3 %}
+## Prerequisites
+ 
+* An Amazon S3 service and bucket.
+* Read/write credentials for the bucket.
+
+
+## Configuration 
+
+In this setup you will need to designate one backup node. The backup node must have read/write access to the S3 compatible storage volume. This node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the storage volume. Nodes are initialized with fallback configs via environment variables, including `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_DEFAULT_REGION`. For more information about the data that is set in the environment variables review the [AWS environment variable configuration documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
```

Comment:
Should we be more explict about the read/write access are corresponed for importers and the exporter, just like the GCP section?

Reply:
Also the proxy traffic part.